ibm/resource_ibm_is_image.go

```diff
@@ -379,7 +505,20 @@ func isWaitForImageAvailable(imageC *vpcv1.VpcV1, id string, timeout time.Durati
 
 	return stateConf.WaitForState()
 }
+func isWaitForImageAvailableForVolume(imageC *vpcv1.VpcV1, id string, timeout time.Duration) (interface{}, error) {
```

Comment:
Why you need different wait function for volume and file?
isWaitForImageAvailable can't we use the same method because we are making a GetImage call